spec.bs

```diff
@@ -817,15 +860,58 @@ On the other hand, methods for getting data from the [=shared storage database=]
                     1. [=list/Append=] |data| to |argumentsList|.
                     1. [=Call=] |operation| with |argumentsList|.
                 1. Otherwise, [=call=] |operation| without any arguments list.
-        1. Resolve |promise| with undefined.
+        1. [=Queue a global task=] on the [=DOM manipulation task source=], given |realm|'s [=global object=], to resolve |promise| with undefined.
+        1. If |options|["`keepAlive`"] is false:
+            1. Wait for |operation| to finish running, if applicable.
```

Comment:
I'm not sure we need to wait here -- I think the [=call=]s above are blocking. Maybe the "[=Queue a global task=] on the [=DOM manipulation task source=], given |realm|'s [=global object=], to resolve |promise| with undefined." needs to be moved earlier?

(ditto for selectURL)

Reply:
Oh I think you're right -- I don't see an equivalent issue there :)